modal_utils/package_utils.py

```diff
@@ -10,7 +10,9 @@
 
 def get_file_formats(module):
     try:
-        endings = [str(p).split(".")[-1] for p in files(module) if "." in str(p)]
+        file_list = files(module)
+        assert file_list is not None
```

Comment:
Haha nice, this type checking would have caught and fixed a bug I just fixed in a separate PR! #1454 👍